Modals/vova_timonin.md

```diff
@@ -0,0 +1,26 @@
+1. Нельзя сидеть сложив руки (with folded hands) и ждать пока все решится само собой.
+You shouldn't sit with folded hands and wait until everything will be decided by itself. 
+
+2. Не нужно меня торопить, я и так не сижу ни секунды.
+You shouldn't hurry up me, I don't sit for a second anyway.
+
+3. Я еще должен покормить собаку и сходить в магазин за продуктами.
+I must feed the dog yet and go to the market for products.
+
+4. Помни, что тебе придется ответить за все изменения в проекте и каждый коммит, который может повлиять на опыт пользователя от использования приложения.
+Remember, you will have to answer about all changes in the project and every commit that can affect to the user expierence of using the application. 
```

Comment:
you will have to keep responsibility for all the changes ... that can affect the user experience in using the application